network/crl_test.go

```diff
@@ -0,0 +1,468 @@
+/*
+Copyright 2020, Cossack Labs Limited
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package network
+
+import (
+	"bytes"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"os"
+	"path"
+	"testing"
+	"time"
+)
+
+const (
+	// Default group, without intermediate certificates
+	TestCertPrefix          = "../tests/ssl"
+	TestCACertFilename      = "ca/ca.crt"
+	TestCertFilename        = "acra-writer/acra-writer.crt"
+	TestRevokedCertFilename = "acra-writer-revoked/acra-writer-revoked.crt"
+	TestCRLFilename         = "crl.pem"
+	TestOCSPCertFilename    = "ocsp-responder/ocsp-responder.crt"
+	TestOCSPKeyFilename     = "ocsp-responder/ocsp-responder.key"
+
+	// TODO add certificates for this group, fix paths, uncomment related code
+	// A different group, uses intermediate CA
+	// TestCertPrefix3          = "../tests/ssl/group-with-intermediate"
+	// TestCACertFilename3      = "ca/ca.crt"
+	// TestICACertFilename3     = "intermediate-ca/intermediate-ca.crt"
+	// TestCertFilename3        = "acra-writer/acra-writer.crt"
+	// TestRevokedCertFilename3 = "acra-writer-revoked/acra-writer-revoked.crt"
+	// TestCRLFilename3         = "intermediate-ca/crl.pem"
+	// TestOCSPCertFilename3    = "intermediate-ocsp-responder/intermediate-ocsp-responder.crt"
+	// TestOCSPKeyFilename3     = "intermediate-ocsp-responder/intermediate-ocsp-responder.key"
```

Comment:
For mass commenting out of code it's more tidy to use `/* block comments */` since they give nicer diffs.

Another alternative is to move all disabled tests into a separate file `crl_with_intermediate_test.go` or something, and then disable its compilation with a line in the beginning of the file:
```go
// +build testIntermediateCA
```
Then at least `gofmt` will do its job, and the diff will be equally tiny.

Though, this has problems of its own, like what to do with all the shared test utilities. Or are they visible to all test files?